draft-ietf-rtcweb-jsep.xml

```diff
@@ -445,9 +445,14 @@ setRemote(ANSWER) |                                   |
           sections when RtpTransceivers are created but not yet
           associated with a m= section, or if RtpTransceivers have been
           stopped and disassociated from m= sections. An RtpTransceiver
-          is never associated with more than one m= section, and once a
-          session description is applied, a m= section is always
-          associated with exactly one RtpTransceiver.</t>
+          is said to be associated with an m= section if its mid property
+          is non-null. Otherwise it is said to be disassociated.
+          The associated m= section is determined using a mapping between
+          transceivers and m= section indices, formed when creating an
+          offer or applying a remote offer. An RtpTransceiver is never
```

Comment:
do you need to spell out how this mapping is created? It's not obvious to me; I was assuming the mapping would always exist, as an array of transceivers that is populated by calls to addTrack/addTransceiver.